### PR TITLE
Allow specifying what fingerprint should be created in the SDK

### PR DESCRIPTION
**Pull request by janderland**

### CODE REVIEW

File: doc.go [hunk 1]
```diff
@@ -6,32 +6,29 @@
 // following link: https://docs.ae.pex.com/installation/, before trying to use
```
Comment (janderland) [inline doc.go:6]: Goland auto formatted a lot of the GoDoc here so sorry for all the changes.

File: doc.go [hunk 2]
```diff
@@ -40,75 +37,79 @@
...
 //
-//     ft, err := pexae.FingerprintBuffer(b)
-//     if err != nil {
-//         panic(err)
-//     }
+// If you only want to use certain types of fingerprinting, you can
+// specify that as well:
+//
+//	ft1, _ := client.FingerprintFileForTypes("/path/to/file.mp4", pexae.FingerprintTypeAudio)
+//	ft2, _ := client.FingerprintBufferForTypes(b, pexae.FingerprintTypeVideo|pexae.FingerprintTypeMelody)
```
Comment (janderland) [inline doc.go:58]: The formatter changed a lot of this GoDoc, but this is the part that is actually new.

File: fingerprint.go [hunk 3]
```diff
@@ -23,19 +33,34 @@ func NewFingerprint(b []byte) *Fingerprint {
...
 func (x *Client) FingerprintFile(path string) (*Fingerprint, error) {
-	return newFingerprint([]byte(path), true)
+	return x.FingerprintFileForTypes(path, FingerprintTypeAll)
+}
+
+// FingerprintFileForTypes is used to generate a fingerprint from a
+// file stored on a disk. The path parameter must be a path to a
+// valid file in supported format. The typ parameter specifies
+// which types of fingerprints to create.
+func (x *Client) FingerprintFileForTypes(path string, typ FingerprintType) (*Fingerprint, error) {
```
Comment (stepanbujnak) [inline fingerprint.go:46]: Just something to consider. We don't really need new function for this, we can define the `FingerprintFile()` function as `FingerprintFile(path string, types ...FingerprintType)` that way you can call it like:

   FingerprintFile(path)

but also

    FingerprintFile(path, FingerprintTypeAudio, FingerprintTypeVideo)

Comment (janderland) [reply fingerprint.go:46]: I never liked this syntax in Go or C cause it allowed you to provided multiple parameters when you should only be providing one.

Comment (janderland) [reply fingerprint.go]: Actually, I changed my mind and made it the way you suggested.

**Review by stepanbujnak — APPROVED** [review]

**Pull request merged**